static/js/components/ProfileProgressControls.js

```diff
@@ -30,15 +30,15 @@ export default class ProfileProgressControls extends React.Component {
     let prevButton, nextButton;
     if(prevStep) {
       prevButton = <button
-        className="mm-button-action prev"
+        className="mm-button action prev"
```

Comment:
It doesn't look like `action` exists anywhere in the CSS, is it declared by material-design?